README.md

```diff
@@ -67,7 +70,7 @@ This component wraps your entire app (but within the DrizzleProvider) and will s
 
 `sendArgs` (object) An object specifying options for the transaction to be sent; namely: `from`, `gasPrice`, `gas` and `value`. Further explanataion of these parameters can be found [here in the web3 documentation](https://web3js.readthedocs.io/en/1.0/web3-eth-contract.html#id19).
 
-`labels` (array) Custom labels; will follow ABI input ordering. Useful for friendlier names. For example "_to" becoming "Recipient Address".
+`labels` (array) Custom labels; will follow ABI input ordering. Useful for friendlier names. For example "\_to" becoming "Recipient Address".
```

Comment:
It was the VC code cleaner, not me personally. I suppose it's not wrong, since in markdown language underscores have a: _meaning_ (italic), so it's good do \_escape\_ them. (see source of this)

I suggest to keep the \\.